full-service/src/error.rs

```diff
@@ -315,6 +315,9 @@ pub enum WalletTransactionBuilderError {
 
     /// Burn Redemption Memo must be exactly 128 characters (64 bytes) long.
     InvalidBurnRedemptionMemo(String),
+
+    /// Error converting a TxOut
+    TxOutConversion(mc_transaction_core::TxOutConversionError),
```

Comment:
This error, as well as a few ones above it, fail to render the argument (they are missing the `: {0}` in the docstring).